<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="loginSCREEN" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>1021</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QLabel" name="labelParagraf">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>360</y>
       <width>501</width>
       <height>151</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Segoe Print&quot;;
color: white</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;br/&gt;Descoperă CODE BUDDY, un loc vibrant pentru a-ți dezvolta abilitățile de programare! Lecții interactive, realizări și o comunitate pasionată te așteaptă să transformi codul în creativitate. Aventura începe acum!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>60</y>
       <width>341</width>
       <height>431</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>100</y>
         <width>228</width>
         <height>191</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="title">
        <string>SignIn</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="labelUser">
            <property name="text">
             <string>email:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="userIN"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="labelPassw">
            <property name="text">
             <string>Password:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="passwIN">
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="loginBTN">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="ToRegisterBTN">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>300</y>
         <width>101</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>TO Register</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QGroupBox" name="groupBox_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>281</width>
         <height>301</height>
        </rect>
       </property>
       <property name="title">
        <string>SingUp</string>
       </property>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>110</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>243</width>
          <height>179</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="labelEmailRegister">
             <property name="text">
              <string>email:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="emailRegisterIN"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="labelUsernameRegister">
             <property name="text">
              <string>username:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="usenameRegisterIN"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="labelPassword2">
             <property name="text">
              <string>password:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="password2IN">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="labelPassword1">
             <property name="text">
              <string>password check:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="Password1IN">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="CreateAccBTN">
           <property name="text">
            <string>Create Account</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QPushButton" name="ToSignIn">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>380</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>To SingIN</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QLabel" name="label_Logo">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>70</y>
       <width>421</width>
       <height>321</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1063</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
